File: backend/video/src/main/java/com/github/makewheels/video2022/etc/app/CheckUpdateResponse.java
package com.github.makewheels.video2022.etc.app;

import lombok.Data;

@Data
public class CheckUpdateResponse {
    private Boolean hasUpdate;
    private Integer versionCode;
    private String versionName;
    private String versionInfo;
    private String downloadUrl;
    private Boolean isForceUpdate;
}

File: android/app/src/main/java/com/github/makewheels/video2022/data/model/CheckUpdateResponse.kt
package com.github.makewheels.video2022.data.model

data class CheckUpdateResponse(
    val hasUpdate: Boolean,
    val versionCode: Int?,
    val versionName: String?,
    val versionInfo: String?,
    val downloadUrl: String?,
    val isForceUpdate: Boolean?
)
